GAE/src/org/waterforpeople/mapping/app/web/rest/dto/SurveyAssignmentDto.java

```diff
@@ -1,68 +0,0 @@
-
-package org.waterforpeople.mapping.app.web.rest.dto;
-
-import java.util.Date;
-import java.util.List;
-
-import com.gallatinsystems.framework.gwt.dto.client.BaseDto;
-
-public class SurveyAssignmentDto extends BaseDto {
```

Comment:
Ahh I see it here now. Just curious how come you didn't modify rather than delete it altogether?

Reply:
I just moved it between packages...